x/btcstaking/keeper/msg_server_test.go

```diff
@@ -1044,7 +1057,9 @@ func FuzzSelectiveSlashing(f *testing.F) {
 		// now BTC delegation has all covenant signatures
 		actualDel, err = h.BTCStakingKeeper.GetBTCDelegation(h.Ctx, stakingTxHash)
 		h.NoError(err)
-		require.True(t, actualDel.HasCovenantQuorums(bsParams.CovenantQuorum))
+		hasQuorum, err := h.BTCStakingKeeper.BtcDelHasCovenantQuorums(h.Ctx, actualDel, bsParams.CovenantQuorum)
```

Comment:
in `legacy/main` branch we had a few more tests for stake expansion, can you add those ?